sdcm/sct_config.py

```diff
@@ -1797,6 +1797,9 @@ def __init__(self):  # noqa: PLR0912, PLR0914, PLR0915
             backend_config_files += self.defaults_config_files[str(backend)]
         self.multi_region_params = self.per_provider_multi_region_params.get(str(backend), [])
 
+        # load docker images defaults
+        self.load_docker_images_defaults()
```

Comment:
won't this override image provided by `SCT_STRESS_IMAGE.cassandra-stress=...`?

Reply:
it's before env variables are evaluated